doc/changelog/10-standard-library/13559-primitive_integers.rst

```diff
@@ -0,0 +1,5 @@
+- **Added:**
+  Library for signed primitive integers, Sint63.
+  (`#13559 <https://github.com/coq/coq/pull/13559>`_,
+  fixes `#12109 <https://github.com/coq/coq/issues/12109>`_,
+  by Ana Borges, Guillaume Melquiond and Pierre Roux).
```

Comment:
Hmm, is it usual to have 2 changelog descriptions for a single PR?

Reply:
I was hoping you (the devs) would tell me. I couldn't decide on whether to put it in the kernel or the stdlib part, so I wrote both. Which do you think makes more sense to keep?

Reply:
If people using signed primitive ints would generally use them through the new library, I think I'd show them under the standard library (while mentioning the kernel additions).  I think the change log is primarily meant to inform users.

Reply:
Ok, thanks.